File: Assets/MAPNAV/Demo Scenes/2D Scene/In-Out.js
#pragma strict

function OnTriggerEnter(other:Collider){
	if(other.tag=="Player"){
		transform.Find("Points").guiText.text="IN";
	}
}
function OnTriggerExit(other:Collider){
	if(other.tag=="Player"){
		transform.Find("Points").guiText.text="OUT";
	}
}

File: Assets/MAPNAV/Demo Scenes/2D Scene/InOut.cs
//MAPNAV Navigation ToolKit v.1.3.2

using UnityEngine;
using System.Collections;

public class InOut : MonoBehaviour 
{
	void OnTriggerEnter(Collider other){
		if(other.tag == "Player"){
			transform.Find("Status").guiText.text = "IN";
		}
	}
	void OnTriggerExit(Collider other){
		if(other.tag == "Player"){
			transform.Find("Status").guiText.text = "OUT";
		}
	}
}